src/helpers/mdObjBucketHelper.ts

```diff
@@ -20,3 +21,9 @@ export function getBucketItems(
 export function isBucketEmpty(buckets: VisualizationObject.IBucket[], bucketName: string): boolean {
     return isEmpty(getBucketItems(buckets, bucketName));
 }
+
+export function getSecondaryIdentifierFromBucketsItem(buckets: VisualizationObject.IBucket[]): string[] {
```

Comment:
I would suggest `getSecondaryMeasuresLocalIdentifiers` as it is not a secondary identifier but a local identifier of the secondary measures.

Reply:
updated :)